services/ymodem/hss_ymodem_loader.c

```diff
@@ -51,9 +51,12 @@
 
 #if IS_ENABLED(CONFIG_SERVICE_QSPI)
 #  include "qspi_service.h"
-//#  include "baremetal/drivers/micron_mt25q/micron_mt25q.h"
```

Comment:
This is my test code of Ymodem, please remove this change

Reply:
changed and verified build